mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +32,72 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Image respond to clicks
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Large Image
+   */
+  isZoomed?: boolean;
+
+  /**
+   * Control Image outside
+   */
+  onOpenChange?: (arg: boolean) => void;
```

Comment:
arg is too abstract naming, choose something more specific)

Reply:
I removed this line because it is no longer needed